src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,21 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+public class ParserTest {
+
+    @Test
+    public void testGetRequest() {

```

Comment:
I don't think this follows our coding standard for naming tests, it should follow the format: `featureUnderTest_testScenario_expectedBehavior()`